plugins/WebsiteMeasurable/MeasurableSettings.php

```diff
@@ -276,19 +282,29 @@ private function makeUseDefaultSiteSearchParams(SitesManager\API $sitesManagerAp
             $field->description .= Piwik::translate('SitesManager_SearchCategoryLabel');
             $field->description .= ': ';
             $field->description .= $searchCategoryGlobal;
-            $field->transform = function () {
-                return null;// never actually save a value for this
+            $field->transform = function ($value) use ($settings) {
+                if ($value) {
+                    $settings->unsetSiteSearchKeywords = true;
```

Comment:
Needed to do that using a class property, as setting a value for the other setting would be directly overwritten afterwards.